tests/sqlparser_common.rs

```diff
@@ -11516,13 +11516,199 @@ fn parse_notify_channel() {
             dialects.parse_sql_statements(sql).unwrap_err(),
             ParserError::ParserError("Expected: an SQL statement, found: NOTIFY".to_string())
         );
-        assert_eq!(
-            dialects.parse_sql_statements(sql).unwrap_err(),
-            ParserError::ParserError("Expected: an SQL statement, found: NOTIFY".to_string())
-        );
     }
 }
 
+#[test]
+fn parse_load_data() {
+    let dialects = all_dialects_where(|d| d.supports_load_data());
+    let only_supports_load_extension_dialects =
+        all_dialects_where(|d| !d.supports_load_data() && d.supports_load_extension());
+    let not_supports_load_dialects =
+        all_dialects_where(|d| !d.supports_load_data() && !d.supports_load_extension());
```

Comment:
if the function is testing the `load_data` feature, it ideally shouldnt care about `supports_load_extension`? e.g. `all_dialects_where(|d| !d.supports_load_data() && d.supports_load_extension());` I imagine should be the same as `all_dialects_where(|d| !d.supports_load_data());`? given that `supports_load_extension` should be a different test function